kafka-ui-react-app/src/components/Connect/Edit/Edit.tsx

```diff
@@ -116,7 +116,7 @@ const Edit: React.FC<EditProps> = ({
           accidentally breaking your connector config!
         </ConnectEditWarningMessageStyled>
       )}
-      <form onSubmit={handleSubmit(onSubmit)}>
+      <form onSubmit={handleSubmit(onSubmit)} aria-label="form">
```

Comment:
We have either to remove `aria-label="form"` or change `aria-label` to something meaningful 
In your case its something like `Connector edit form` (I'm not sure on this text)

Reasoning: Aria label has to be meaningful an unique 
[MDN role form](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/form_role)
[MDN aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)